applications.yaml

```diff
@@ -702,9 +710,14 @@
   identifier:
     kind: Class
     id: TeamsWebView
-    matching_strategy: Legacy
+    matching_strategy: Equals
   options:
   - tray_and_multi_window
+  - object_name_change
+  float_identifiers:
+  - kind: Class
+    id: TeamsWebView
+    matching_strategy: Equals
```

Comment:
All MS Teams windows will float if this is applied,

Reply:
I didn't find anyway to actually match for only teams call windows. I agree, after one day of test that this is annoying to just put everthing floating. I'll remove this hunk